src/main/default/classes/RatingsHandler.cls

```diff
@@ -1,6 +1,6 @@
 public with sharing class RatingsHandler {
     @AuraEnabled(cacheable=false)
-    public static List<Rating__c> getRating(String documentId) {
+    public static Rating__c getRating(String documentId) {
```

Comment:
The SOQL query is returning a list. You should check the number of records and return only the first one.

Reply:
Improved👌